apps/common-app/src/examples/LayoutAnimations/LayoutTransitionExample.tsx

```diff
@@ -0,0 +1,295 @@
+  TouchableOpacity,
+} from 'react-native';
+import Animated, {
+  LinearTransition,
+  SequencedTransition,
+  FadingTransition,
+  FadeOut,
+  JumpingTransition,
+  CurvedTransition,
+  EntryExitTransition,
+  FlipInEasyX,
+  FlipOutYRight,
+} from 'react-native-reanimated';
+import { createStackNavigator } from '@react-navigation/stack';
+import { NavigationContainer } from '@react-navigation/native';
+import { type StackScreenProps } from '@react-navigation/stack';
+
+interface Item {
+  id: number;
+  content: string;
+  color: string;
+}
+
+const INITIAL_LIST = [
+  { id: 1, content: '🍌', color: '#b58df1' },
+  { id: 2, content: '🍎', color: '#ffe780' },
+  { id: 3, content: '🥛', color: '#fa7f7c' },
+  { id: 4, content: '🍙', color: '#82cab2' },
+  { id: 5, content: '🍇', color: '#fa7f7c' },
+  { id: 6, content: '🍕', color: '#b58df1' },
+  { id: 7, content: '🍔', color: '#ffe780' },
+  { id: 8, content: '🍟', color: '#b58df1' },
+  { id: 9, content: '🍩', color: '#82cab2' },
+];
+
+type RootStackParamList = {
+  Home: undefined;
+  linear: { title: string; transition: any };
+  fading: { title: string; transition: any };
+  sequenced: { title: string; transition: any };
+  jumping: { title: string; transition: any };
+  curved: { title: string; transition: any };
+  entryexit: { title: string; transition: any };
+};
+
+type HomeScreenProps = StackScreenProps<RootStackParamList, 'Home'>;
+
+function HomeScreen({ navigation }: HomeScreenProps) {
+  return (
+    <View style={styles.homeContainer}>
+      <Button
+        title={'Linear Transition'}
+        onPress={() =>
+          navigation.navigate('linear', {
+            title: 'LinearTransition',
+            transition: LinearTransition,
+          })
+        }
+      />
+      <Button
+        title="Sequenced Transition"
+        onPress={() =>
+          navigation.navigate('sequenced', {
+            title: 'SequencedTransition',
+            transition: SequencedTransition,
+          })
+        }
+      />
+      <Button
+        title="Fading Transition"
+        onPress={() =>
+          navigation.navigate('fading', {
+            title: 'FadingTransition',
+            transition: FadingTransition,
+          })
+        }
+      />
+      <Button
+        title="Jumping Transition"
+        onPress={() =>
+          navigation.navigate('jumping', {
+            title: 'JumpingTransition',
+            transition: JumpingTransition,
+          })
+        }
+      />
+      <Button
+        title="Curved Transition"
+        onPress={() =>
+          navigation.navigate('curved', {
+            title: 'CurvedTransition',
+            transition: CurvedTransition,
+          })
+        }
+      />
+      <Button
+        title="Entry/Exit Transition"
+        onPress={() =>
+          navigation.navigate('entryexit', {
+            title: 'EntryExit',
+            transition:
+              EntryExitTransition.entering(FlipInEasyX).exiting(FlipOutYRight),
+          })
```

Comment:
"Why do we need to define those parameters twice - here and as `initialParams`? Is there a way to simplify this?"